UIBarButtonItem+Action.swift

```diff
@@ -0,0 +1,92 @@
+import UIKit
+import RxSwift
+import RxCocoa
+import ObjectiveC
+
+public extension UIBarButtonItem {
+	
+	/// Binds enabled state of action to button, and subscribes to rx_tap to execute action.
+	/// These subscriptions are managed in a private, inaccessible dispose bag. To cancel
+	/// them, set the rx_action to nil or another action.
+	public var rx_action: CocoaAction? {
+		get {
+			var action: CocoaAction?
+			doLocked {
+				action = objc_getAssociatedObject(self, &AssociatedKeys.Action) as? Action
+			}
+			return action
+		}
+		
+		set {
+			doLocked {
+				// Store new value.
+				objc_setAssociatedObject(self, &AssociatedKeys.Action, newValue, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
+				
+				// This effectively disposes of any existing subscriptions.
+				self.resetActionDisposeBag()
+				
+				// Set up new bindings, if applicable.
+				if let action = newValue {
+					action
+						.enabled
+						.bindTo(self.rx_enabled)
+						.addDisposableTo(self.actionDisposeBag)
+					
+					// Technically, this file is only included on tv/iOS platforms,
+					// so this optional will never be nil. But let's be safe 😉
+					let lookupControlEvent: ControlEvent<Void>?
+					
+					#if os(tvOS)
+						lookupControlEvent = self.rx_primaryAction
```

Comment:
I actually don't see `rx_primaryAction` on the `UIBarButtonItem` Rx extension, so this would actually not compile for tvOS targets. This is a problem with our CI setup – we only run unit tests against one platform, probably worth opening an issue. Thoughts?